Firmware/lcd.cpp

```diff
@@ -776,6 +787,7 @@ void lcd_buttons_update(void)
 	if (lcd_buttons & EN_B) enc |= B10;
 	if (enc != lcd_encoder_bits)
 	{
+		lcd_backlight_wake_trigger = true; // flag event, knob rotated
```

Comment:
I'm not sure this is entirely correct. I feel like this will make the backlight wake even if the encoder was moved only 1/4 of a tick of rotation. That might be enough to create random wakes of the backlight if the knob is between two encrot steps and oscillates between them due to noise/vibrations.

Reply:
That’s a good point. 👍

Reply:
Resolved with [8a926fa](https://github.com/prusa3d/Prusa-Firmware/pull/4010/commits/8a926fad103b1409701a092b98f5137ceff6ad5e)